### PR TITLE
Feature/pc parts scraper seed

### PR DESCRIPTION
**Pull request by AdrianIlaga**
This PR covers the addition of the pc parts scraper in the seeds.rb file in order to populate the database with data. In addition, it also covers a fix to the database as well as the scraper function

Seeds.rb

Added Scrape function for various pc parts in the seed file

image

Output
image

Data
image

Scraper.rb

Edited code to allows the scraper to access succeeding pages, not just the first page of the website

Database

Removed "type" column from ssds table. It was not supposed to be there.

### CODE REVIEW

File: app/services/scraper.rb [hunk 1]
```diff
@@ -2,27 +2,30 @@
...
     products = response.xpath("//a[contains(@class, 'productItemLink')]")
-    return if products.empty?
+
+    return unless products.any?
 
     products.each do |a|
       request_to :parse_repo_page, url: absolute_url(a[:href], base: url)
     end
-    @page += 1
+    @@page += 1
```
Comment (jarencudilla) [inline app/services/scraper.rb:27]: This is genius!

File: db/schema.rb [hunk 2]
```diff
@@ -223,25 +223,10 @@
     t.string "memory_size"
   end
 
-  create_table "ssd_tables", force: :cascade do |t|
```
Comment (jarencudilla) [inline db/schema.rb:226]: Thanks for removing this, Not sure how this got generated.

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,6 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+# Scraper Seeds
+parts = [Ssd, Hdd, CaseFan, Case, CpuFan, Cpu, Gpu, Mobo, Psu, Ram]
```
Comment (jarencudilla) [inline db/seeds.rb:2]: YEY! Now I can start with our system configurator

**Review by jarencudilla — APPROVED** [review]

**Review by EliezerAmbion — COMMENTED** [review]
Approved! Thanks for this.

**Pull request merged**